test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -511,25 +513,27 @@ public void PreferenceSetting()
 
             initalSetting.ConnectorType = ConnectorType.BEZIER;
             initalSetting.ConsoleHeight = 100;
-            initalSetting.IsBackgroundPreviewActive = true;
+            initalSetting.SetIsBackgroundPreviewActive(backgroundPreviewName, true);
 
             initalSetting.Save(tempPath);
```

Comment:
I don't see any tests that attempt to load a current format dynSettings file... what happens then?